app_radio_module/src/main.c

```diff
@@ -20,15 +21,63 @@ LOG_MODULE_REGISTER(main, CONFIG_APP_RADIO_MODULE_LOG_LEVEL);
 K_QUEUE_DEFINE(lora_tx_queue);
 K_QUEUE_DEFINE(net_tx_queue);
 
+// Threads
+static K_THREAD_STACK_DEFINE(udp_rx_stack, UDP_RX_STACK_SIZE);
+static struct k_thread udp_rx_thread;
+
 // Callbacks
 GNSS_DATA_CALLBACK_DEFINE(DEVICE_DT_GET(DT_ALIAS(gnss)), l_gnss_data_debug_cb);
 GNSS_SATELLITES_CALLBACK_DEFINE(DEVICE_DT_GET(DT_ALIAS(gnss)), l_gnss_debug_sat_count_cb);
 
+// Devices
 static const struct gpio_dt_spec led0 = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
 static const struct gpio_dt_spec led1 = GPIO_DT_SPEC_GET(LED1_NODE, gpios);
 static const struct device *const lora_dev = DEVICE_DT_GET_ONE(semtech_sx1276);
 static const struct device *const wiznet = DEVICE_DT_GET_ONE(wiznet_w5500);
 
+// Networking
+#define UDP_RX_BUFF_LEN 256 // TODO: Make this a KConfig
+#define NUM_SOCKETS 2
+static uint8_t udp_rx_buffer[UDP_RX_BUFF_LEN];
+static int udp_sockets[NUM_SOCKETS] = {0};
+static int udp_socket_ports[NUM_SOCKETS] = {10000};
+static l_udp_socket_list_t udp_socket_list = {
+        .sockets = udp_sockets,
+        .num_sockets = 2
+};
+
+static void udp_rx_task(void *socks, void *buff_ptr, void *buff_len) {
+    l_default_receive_thread(socks, buff_ptr, buff_len);
+}
+
+static void init_networking() {
+    if (l_check_device(wiznet) != 0) {
+        LOG_ERR("Wiznet device not found");
+        return;
+    }
+
+    int ret = l_init_udp_net_stack("192.168.144.81");
+    if (ret != 0) {
+        LOG_ERR("Failed to initialize UDP networking stack: %d", ret);
+        return;
+    }
+
+    for (int i = 0; i < udp_socket_list.num_sockets; i++) {
+        udp_socket_list.sockets[i] = l_init_udp_socket("192.168.144.81", udp_socket_ports[i]);
```

Comment:
Magic number? Put this in a kconfig or a define.

Reply:
Yep was gonna change that. Its now #define in backplane_defs so other modules know which IP to talk to. I still need to fully flesh out the network architecture